npm/npm_import.bzl

```diff
@@ -57,11 +277,39 @@ def npm_translate_lock(name, npm_package_lock = None, yarn_lock = None, pnpm_ver
             version = pnpm_version,
         )
 
-    npm_translate_lock_rule(
+    # convert bins to a string_list_dict to satisfy attr type in repository rule
+    bins_string_list_dict = {}
+    if type(bins) != "dict":
+        fail("Expected bins to be a dict")
+    for key, value in bins.items():
+        if type(value) != "dict":
+            fail("Expected values in bins to be a dicts")
+        for value_key, value_value in value.items():
+            if key not in bins_string_list_dict:
```

Comment:
This could be up a line outside the loop

Reply:
Nice catch